### ISSUE TITLE
Build failing with current nightly

### ISSUE THREAD

**Issue by johanneskoester** [BurntSushi/rust-csv#26]
With current rust, rust-csv does not build:
```
/home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/csv-0.13.2/src/reader.rs:172:37: 172:51 error: the trait `std::ffi::AsOsStr` is not implemented for the type `P` [E0277]

/home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/csv-0.13.2/src/reader.rs:172 Ok(Reader::from_reader(try!(fs::File::open(path))))

^~~~~~~~~~~~~~

<std macros>:1:1: 6:57 note: in expansion of try!

/home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/csv-0.13.2/src/reader.rs:172:32: 172:59 note: expansion site

/home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/csv-0.13.2/src/writer.rs:89:37: 89:53 error: the trait `std::ffi::AsOsStr` is not implemented for the type `P` [E0277]

/home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/csv-0.13.2/src/writer.rs:89 Ok(Writer::from_writer(try!(fs::File::create(path))))

^~~~~~~~~~~~~~~~

<std macros>:1:1: 6:57 note: in expansion of try!

/home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/csv-0.13.2/src/writer.rs:89:32: 89:61 note: expansion site

error: aborting due to 2 previous errors

Could not compile `csv`.
```

**Comment by BurntSushi** [comment]
OK, should be fixed in `0.13.3`. Just run `cargo update`. :-)

**Comment by johanneskoester** [comment]
Thank you! Awesome library!
